src/components/contact.tsx

```diff
@@ -0,0 +1,377 @@
+  position: absolute;
+  top: 14px;
+  left: 8px;
+  z-index: 1001;
+  color: ${(props) =>
+    props.dark ? "white" : "black"}; // Changes the color based on the dark prop
+  img {
+    fill: currentColor; // Makes the SVG inherit the color property
+  }
+`;
+
+const WrapperContent = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  transform: scale(0.93);
+  // margin-top: 232px;
+  margin-top: 35px;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    margin-top: 28px;
+  }
+
+  @media (max-width: 600px) {
+    transform: scale(0.95);
+    margin-top: 28px;
+  }
+
+  @media (max-width: 500px) {
+    transform: scale(0.96);
+    margin-top: 28px;
+  }
+
+  @media (max-width: 375px) {
+    transform: scale(0.95);
+    margin-top: 28px;
+  }
+`;
+
+const StyledBoxContentWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledBox = styled.div`
+  background: ${colors.white};
+  margin-top: 24px;
+  padding: 40px;
+  border-radius: 20px;
+  max-width: 600px;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  justify-content: center;
+  overflow: hidden;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    max-width: 100%;
+    padding: 30px;
+  }
+  @media all and (max-width: ${breakpoints.sm}px) {
+    max-width: 100%;
+    padding: 25px;
+  }
+  @media (max-width: 375px) {
+    margin-top: 8px;
+    overflow: auto;
+    max-height: 80vh;
+    margin-top: 30px;
+  }
+`;
+
+const StyledHeading = styled.p`
+  font-size: 36px;
+  font-weight: 430;
+  line-height: 42px;
+  color: ${colors.black};
+  letter-spacing: 1px;
+  opacity: 0.87;
+  margin: 0px 0px 30px 0px;
+`;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+`;
+
+const InputWrapper = styled.div`
+  margin: 0px;
+  position: relative;
+`;
+
+const StyledInput = styled.input`
+  margin: 0px;
+  width: 100%;
+  height: 63px;
+  width: 515px;
+  border-radius: 12px;
+  padding: 20px;
+  background-color: rgba(255, 255, 255, 1);
+  color: ${colors.black};
+
+  &:focus {
+    outline: none;
+  }
+  @media all and (max-width: ${breakpoints.md}px) {
+    width: 100%;
+  }
+`;
+
+const Label = styled.label`
+  margin-left: 10px;
+  position: absolute;
+  left: 0.6rem;
+  top: 0.6rem;
+  transition: all 0.3s ease;
+  pointer-events: none;
+  background-color: transparent;
+  color: ${colors.black};
+
+  &.active {
+    top: -2px;
+    left: 11px;
+    font-size: 0.75rem;
+  }
+`;
+
+const StyledIcon = styled(ArrowUpRight)`
+  height: 30px;
+  margin-left: 28px; */
+
+  transition: 0.3s ease;
+
+`;
+const StyledButton = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  padding: 20px;
+  margin-top: 16px;
+  border-radius: 26px;
+  width: 313px;
+  margin-left: auto;
+  margin-right: auto;
+  height: 79px;
+  color: rgba(29, 29, 29, 1);
+  font-weight: 100;
```

Comment:
Here is the font problem! It wasn't able to load a font-weight of 100 because that doesn't exist, therefore it would default to the backup font

```suggestion
  font-weight: 500;
```